src/Core/Pages/Accounts/LockPage.xaml.cs

```diff
@@ -233,6 +233,7 @@ private async Task UnlockedAsync()
             }
             var previousPage = await AppHelpers.ClearPreviousPage();
 
+            _appOptions.HasJustLoggedInOrUnlocked = true;
```

Comment:
💭 Instead of having this variable scattered around and risk missing a path to the vault, can't we just check once on TabsPage.OnCreate with a control variable `HasRunCheckOrganizationUnassignedItems`?

Reply:
The problem is that `TabsPage.OnCreate` can be called when the vault has timeout never, then it's not technically after login/unlock as it will be called when opening the app and given timeout never the vault would be already unlocked.